stdlib/public/Concurrency/TaskLocal.swift

```diff
@@ -160,6 +164,7 @@ public final class TaskLocal<Value: Sendable>: Sendable, CustomStringConvertible
   /// If the value is a reference type, it will be retained for the duration of
   /// the operation closure.
   @discardableResult
+  @inline(__always)
```

Comment:
Why do we need this?

Reply:
Was John’s suggestion to add those

Reply:
I over did perhaps, removing all except the one we need.

Reply:
Just added inlinable to the `with...` methods now, I think that's what @rjmccall had in mind?